### PR TITLE
minor typos and grammar/style changes

### PR DESCRIPTION
**Pull request by polypmer**
mostly typos and some subtle style/grammar changes.

In the **second paragraph**, the subject for "provide" could possibly be the *method* and not the *programmer* ("you"). Maybe it is more general or accurate to say that the "author must provide" (rather than "write a method which provides") a means for inputing the verification string. Otherwise I would change the sentence to (with the method as subject):

"You must write a method which takes a URL that the user needs to visit and provides
some way for that user to input a string value, which you then give to PyPump."

(I figure this process could be split in two methods? But it'll generally be in one?)

In the **first paragraph**, I am not literate in the OAuth protocol, so my edits could very well be wrong (although there is something off going on with subject-verb agreements). I notice the procedure is very similar to the second paragraph, maybe the two paragraphs could be merged? Or maybe the first paragraph rather a higher level explanation?

### CODE REVIEW

File: docs/gettingstarted/verifier_callback.rst [hunk 1]
```diff
@@ -2,15 +2,15 @@
 Getting Verifier
 ================
 
-As part of OAuth to allow OOB (Out of band) applications to have access to the account
-we have a link that they will click, follow the instructions and then copy the verifier
-into the application which we then relay to the server with other tokens. The server
-will them provide us with the credentials that we can use.
+For OAuth to allow OOB (Out of band) applications to have access to an account,
+we must provide a link, instructions, and a means of copying the verifier into an application
+and relaying it to the server with other tokens. The server
```
Comment (moggers87) [inline docs/gettingstarted/verifier_callback.rst]: > … and a means of copying … and relaying it

One too many  ands?

**Comment by moggers87** [comment]
Other than my line note, looks good to me.

Assigning to @xray7224 to make sure the OAuth stuff is OK.

File: docs/gettingstarted/verifier_callback.rst (hunk 1 above)
Comment (polypmer) [inline docs/gettingstarted/verifier_callback.rst]: yeah, I agree. I was trying to go for "the means of copying... and relaying" from the list of things one must "provide" (links, instructions, and means). I think its grammatically correct albeit confusing. And, anyway, too many ands for my taste.  Maybe it bares repetition? Something like, 

+we must provide a link, instructions, a means of copying the verifier into an application,
+and **a means** of relaying it to the server with other tokens.

**OR!** I'm thinking splitting it into what the user needs and what the application/server needs:

+First, we must provide a link and instructions for the user. Then we must provide a means of copying the verifier into an application and relaying it to the server with other tokens.

Or is this all too silly/irrelevant anyway?

**Comment by xray7224** [comment]
OAuth stuff is okay :) Read through it and it makes sense. Thanks for the changes!

File: docs/gettingstarted/verifier_callback.rst (hunk 1 above)
Comment (moggers87) [inline docs/gettingstarted/verifier_callback.rst]: I like the second one better. It's nicely broken up into two sentences.

**Comment by moggers87** [comment]
Sorry, didn't see your update. Thanks for the PR :)

**Pull request merged**